.circleci/config.yml

```diff
@@ -296,7 +325,15 @@ workflows:
       #        ignore: /.*/
       #      tags:
       #        only: /^v\d+\.\d+\.\d+-?.*/
-      - test-integration:
+      - test-integration-fs:
+          requires:
+            - build
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v\d+\.\d+\.\d+-?.*/
```

Comment:
Why is this?

Reply:
Let's see... This is part of the publish-release pipeline and we only want to run this if we're on a version tag. We allow for pre-releases in waiting for semver pre-releases to be supported by the marketplace. Which maybe will never happen.

* https://github.com/microsoft/vscode-vsce/issues/148
* https://github.com/microsoft/vscode/issues/15756#issuecomment-916728348

It's copied over from Calva where more of the pre-release publishing is prepared for. We could consider cleaning it away from here.